contracts/Peeranha.sol

```diff
@@ -88,6 +88,9 @@ contract Peeranha is IPeeranha, Initializable {
      * - Must be an existing user.  
      */
     function updateUser(address userAddress, bytes32 ipfsHash) external override {
+        if (!UserLib.isExists(userContext.users, msg.sender)) {
+            UserLib.create(userContext.users, msg.sender, bytes32(0xf5cd5e9d6332d6b2a532459dfc262f67d4111a914d00edb7aadd29c30d8ac322));
```

Comment:
Move hash `0xf5cd5e9d6332d6b2a532459dfc262f67d4111a914d00edb7aadd29c30d8ac322` to constants